### PR TITLE
Add first set of tests for the upcoming HPC-Module

### PR DESCRIPTION
**Pull request by Soulofdestiny**
related progress item: https://progress.opensuse.org/issues/13750
related FATE: https://fate.suse.com/320596

The description of testcases for that module is still in a kind of development, but these testcases are the very base for validating the product (e.g. catching dependency issues)

### CODE REVIEW

File: tests/hpc/cpuid.pm [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
```
Comment (okurz) [inline tests/hpc/cpuid.pm]: only 2016

File: tests/hpc/install.pm [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
```
Comment (okurz) [inline tests/hpc/install.pm]: only 2016

File: tests/hpc/rasdaemon.pm [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
```
Comment (okurz) [inline tests/hpc/rasdaemon.pm]: …

File: tests/hpc/rasdaemon.pm [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils qw/wait_boot/;
+
+sub run() {
+    my $self = shift;
+
+    # TODO
+    # See https://fate.suse.com/318824
```
Comment (okurz) [inline tests/hpc/rasdaemon.pm]: Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

**Comment by okurz** [comment]
good first step

File: tests/hpc/rasdaemon.pm (hunk 4 above)
Comment (Soulofdestiny) [inline tests/hpc/rasdaemon.pm]: yeah... typical copy&paste fails
Thanks for review

**Comment by Soulofdestiny** [comment]
fixed

**Pull request merged**